code/game/objects/items/oddities.dm

```diff
@@ -421,6 +439,18 @@
 		STAT_MEC = 7
 	)
 
+/obj/item/oddity/common/book_unholy/opened
+	name = "awakened unholy book"
+	desc = "The book floats open in your hands, infinite forbidden knowledge and non-euclidean geometry contained within at your disposal. \
+			The anomaly has been strengthened in its odd nature by forces unknown, but is still perfectly functional for your rituals..."
+	icon_state = "book_skull_open"
+	item_state = "book_skull_open"
+	oddity_stats = list(
+		STAT_COG = 6,
+		STAT_MEC = 9
+	)
+	perk = /datum/perk/scribe // The book will render you BLIND!
```

Comment:
Why do these force a perk?

Reply:
So that you could either learn the perks through ritual or book ascension, considering the spell to "ascend" them is the only way to obtain these
If it's too much for balance I can nix the perks, I don't mind